app/views/podcasts/show.rss.builder

```diff
@@ -74,11 +74,7 @@ xml.rss 'xmlns:atom' => 'http://www.w3.org/2005/Atom',
     xml.sy :updateFrequency, @podcast.update_frequency if @podcast.update_frequency
     xml.sy :updateBase, @podcast.update_base if @podcast.update_base
 
-    episode_count = @podcast.display_episodes_count.to_i
```

Comment:
there were 2 attributes enforcing this, one in the view, one in the model - now just using one in the model